week3/Week3HomeWork/Controllers/ConverterController.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Week3HomeWork.Models;
+
+namespace Week3HomeWork.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ConverterController : ControllerBase
+    {
+        [HttpGet("GallonsToLitres")]
+        public double GallonsToLitres(int Litres)
+        {
+            return 0.264 * Litres;
+        }
+
+        [HttpGet("ConvertMiles")]
+        public Distance ConvertMiles(int miles)
+        {
+            return new Distance { Miles = miles, Kilometers = miles * 1.609 };
+        }
+
+        [HttpPost("convert")]
+        public List<ConversionResponse> ConvertValues([FromBody] ConversionRequest Request)
+        {
+            return new ConversionResponse().conversions(Request);
```

Comment:
Method names should start with uppercase, `Conversions` in this case, and variables with lowercase `request`

Reply:
Done